pkg/backup/backup.go

```diff
@@ -214,17 +212,16 @@ type VolumeSnapshotterGetter interface {
 }
 
 // Backup backs up the items specified in the Backup, placing them in a gzip-compressed tar file
-// written to backupFile. The finalized api.Backup is written to metadata.
-func (kb *kubernetesBackupper) Backup(logger logrus.FieldLogger, backupRequest *Request, backupFile io.Writer, actions []velero.BackupItemAction, volumeSnapshotterGetter VolumeSnapshotterGetter) error {
+// written to backupFile. The finalized api.Backup is written to metadata. Any error returned by
+// this function causes the backup to be Failed.
```

Comment:
Maybe instead of `Any error returned by this function causes the backup to be Failed`, suggestion:

`Any error that constitutes a complete backup failure is returned. Errors that represent a failure that is considered partial, ie which means the backup can still be completed, are logged.`

Or any modification that clarifies why some errors are not returned.